src/computerAI.js

```diff
@@ -1,5 +1,7 @@
 import {gameBoard} from "./gameBoard";
```

Comment:
This is a bit from my first attempt at a smart computer.  No need to review this file, but I will talk about my mistakes a bit more during our meeting.